oemof/groupings.py

```diff
@@ -9,15 +9,17 @@
 SPDX-License-Identifier: GPL-3.0-or-later
 """
 
-try:
-    from collections.abc import (Hashable, Iterable, Mapping,
-                                 MutableMapping as MuMa)
-except ImportError:
-    from collections import (Hashable, Iterable, Mapping,
+from collections.abc import (Hashable, Iterable, Mapping,
                              MutableMapping as MuMa)
 from itertools import chain, filterfalse
 
 
+# TODO: Update docstrings.
```

Comment:
Is this done? They seem to use nodes, now.

Reply:
Eight lines later, `Entity` is still used. So that `TODO` can unfortunately not be ticked off yet.